Quicklogic upstream rebase

lnsharma opened the pull request:
Adding 3 new packages to device xml file for EOS-S3.

kgugala commented:
I'm afraid this PR is addressed incorrectly. Since the `quicklogic-upstream-rebase` branch is our working branch, the PR should target it. Right now the changes are already on the `quicklogic-upstream-rebase` branch and PR wants to merge them to master.

Since the changes are just adding some test cases, they should not break anything. I'll close this PR as it is not really required here anymore.

lnsharma commented:
@kgugala I added 3 new package definition in QLAL4S3B.xml device file and this checkin is not for addition of any test case. Today, I updated rebase branch and did not find the changes. I initially added these 3 package definition in https://github.com/QuickLogic-Corp/EOS-S3.git and then wants to merge them into rebase branch.
